server/models/user.js

```diff
@@ -1,76 +1,24 @@
 import mongoose from 'mongoose';
-import httpStatus from 'http-status';
-import APIError from '../helpers/APIError';
 
-/**
- * User Schema
- */
+
 const UserSchema = new mongoose.Schema({
-  username: {
-    type: String,
-    required: true
-  },
-  mobileNumber: {
+  name: {
     type: String,
     required: true,
-    match: [/^[1-9][0-9]{9}$/, 'The value of path {PATH} ({VALUE}) is not a valid mobile number.']
   },
+  email: String,
+  auth: String,
+  escalationPolicy: { 
+    type: mongoose.Schema.Types.ObjectId, 
```

Comment:
This will create a separate document for the escalation policy and store the ObjectId, we want to fully embed the document using [this approach](http://mongoosejs.com/docs/2.7.x/docs/embedded-documents.html)